tests/losses/loss_functions_test.py

```diff
@@ -48,6 +82,17 @@ def test_sisdr(n_src, function_triplet):
     )
 
 
+@pytest.mark.parametrize("loss", loss_properties)
+def test_sisdr_and_mse_shape_checks(loss):
+    pairwise, singlesrc, multisrc, arbitrary_last_dim = loss
+    assert_loss_checks_shape(pairwise, (3, 1000), arbitrary_last_dim)
+    assert_loss_checks_shape(singlesrc, (1000,), arbitrary_last_dim)
+    # Special case for multisrc_mse that shares the same code with
+    # singlesrc_mse and thus accepts 1-dim tensors.
+    no_batch_ok = multisrc == mse.multisrc_mse
+    assert_loss_checks_shape(multisrc, (3, 1000), arbitrary_last_dim, no_batch_ok)
```

Comment:
I don't see the purpose of the `no_batch_ok`.
Without batch, the `multisrc_mse` will return a tensor of size `n_src`. It's not really what we want.

Is there something I'm missing?

Reply:
If we don't add the special case, the checks will fail: It will assert that `multisrc_mse` fails with `(3, 1000)` shaped input. But since `MultiSrcMSE = SingleSrcMSE`, this is identical to testing `singlesrc_mse` with `(1000,)` input

Reply:
I see now, thanks. 
I'm not convinced by how useful it is to check that it doesn't work without batch, but it's fine.